src/components/TabNav/TabNav.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import { ArgsTable, Canvas, Props, Story } from '@storybook/addon-docs/blocks';
+import { TabNav, TabNavItem } from './TabNav';
+
+<Meta title="TabNav" component={TabNav} />
+
+# TabNav and TabNavItem
+
+Navigational components for linking to sibling pages. Use `TabNav` for the container element and `TabNavItem` for individual navigational items.
+
+A link (`<a>`) or button (`<button>`) element should be passed into each `TabNavItem` as a child node.
+
+## Usage
+
+```js
+import { TabNav, TabNavItem } from '@quartz/interface/src/components';
```

Comment:
i've been leaving this out. feels pretty redundant. i have a pr i'll put up to include it in the welcome story